Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM python:3.5-alpine
+# Using debian buster because this: https://pythonspeed.com/articles/base-image-python-docker-images/
+FROM python:3.7-slim-buster
 
 WORKDIR /app
 COPY clinical_trials_api_scraper /app/clinical-trials-api-scraper/clinical_trials_api_scraper
 COPY setup.py /app/clinical-trials-api-scraper
 
 RUN pip install /app/clinical-trials-api-scraper
 
-#Set up cron
-COPY daily-trials-scraping /etc/cron.d/daily-trials-scraping
-RUN chmod 0644 /etc/cron.d/daily-trials-scraping
-RUN crontab /etc/cron.d/daily-trials-scraping
-
 # Run scraper once to bootstrap db then set up cron
-CMD update_trials_store.py --max-id 100 && crond -f -l 8
+CMD update_trials_store.py --max-id 100 --host ${GRAPHQL_HOST} --port ${GRAPHQL_PORT} && sleep ${SECONDS_BETWEEN_SCRAPES}
```

Comment:
Oh, so the idea is to have this succeed and rely on a restart: always setup in the docker-compose file. I'm not sure I love this as a final way to do this (we'll want it to run once a day precisely and I think having docker containers not rely on ending and restarting is better) but might be better for now and add in cron later?

Reply:
Yeah, I'm not super happy with this, but it's very simple at least and can serve for now. I read some stuff about cron in docker not being great (apparently it can be hard to do logging right and debug issues), and I didn't find an elegant way to pass Docker variables to the command via cron.